oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RunningRule.java

```diff
@@ -343,6 +345,38 @@ private boolean isMatch() {
                                 break;
                         }
                         break;
+                    case MULTI_INTS:
+                        int[] ivalueArray = ((MultiIntValuesHolder)metrics).getValues();
+                        Integer[] iaexpected = RunningRule.this.threshold.getIntValuesThreshold();
+                        MULTI_VALUE_CHECK:
+                        for (int i = 0; i < ivalueArray.length; i++) {
+                            ivalue = ivalueArray[i];
+                            Integer iNullableExpected = 0;
+                            if (iaexpected.length > i) {
+                                iNullableExpected = iaexpected[i];
+                                if (iNullableExpected == null) {
+                                    continue;
+                                }
+                            }
+                            switch (op) {
+                                case LESS:
+                                    if (ivalue < iNullableExpected) {
+                                        matchCount++;
+                                        break MULTI_VALUE_CHECK;
+                                    }
```

Comment:
Missing `break` after `if` statement

```suggestion
                                    }
                                    break;
```